crates/config/src/sections/experimental.rs

```diff
@@ -106,3 +114,10 @@ impl ExperimentalConfig {
 impl ConfigurationSection for ExperimentalConfig {
     const PATH: Option<&'static str> = Some("experimental");
 }
+
+/// Configuration options for the inactive session expiration feature
+#[derive(Clone, Debug, Deserialize, JsonSchema, Serialize)]
+pub struct SessionLimitConfig {
+    pub soft_limit: u64,
+    pub hard_limit: u64,
```

Comment:
TBD, how does it behave when a limit is set to zero? Would it be worth using `NonZeroU64`?